fonts/config.json

```diff
@@ -1,4 +1,18 @@
 [
+  {
+    "family": "Source Sans 3",
+    "path": "SourceSans3-Regular.ttf",
+    "weight": 400,
+    "style": "NORMAL",
+    "subset": false
+  },
+  {
+    "family": "Source Sans 3",
+    "path": "SourceSans3-Bold.ttf",
+    "weight": 700,
+    "style": "NORMAL",
+    "subset": false
+  },
   {
```

Comment:
Må beholde de gamle fontene siden de brukes av malene som padm2 bruker for journalføring av innkommende dialogmeldinger.